src/qudi/logic/scanning_optimize_logic.py

```diff
@@ -389,3 +446,19 @@ def _set_default_scan_settings(self):
         self._scan_frequency = {ax.name: max(ax.frequency.minimum, ax.frequency.maximum / 100) for ax in axes.values()}
         self._back_scan_resolution = {}
         self._back_scan_frequency = {}
+
+    def _set_scan_sequence(self):
```

Comment:
Ambigous name of function; it's not a setter. Maybe _set_default_scan_sequence()?

Reply:
changed the function name